controllers/argocd/util_test.go

```diff
@@ -1384,3 +1384,49 @@ func TestAppendUniqueArgs(t *testing.T) {
 		})
 	}
 }
+
+func TestRemoveNamespaceManagementCRs(t *testing.T) {
```

Comment:
What is this test testing?

Reply:
This needs to be removed from last review itself, which I missed, i will remove it now